test/image/mocks/image_zmin_zmax.json

```diff
@@ -60,6 +60,14 @@
   "layout": {
     "width": 400,
     "height": 800,
+    "yaxis": {"scaleanchor": null},
+    "yaxis2": {"scaleanchor": null},
+    "yaxis3": {"scaleanchor": null},
+    "yaxis4": {"scaleanchor": null},
+    "yaxis5": {"scaleanchor": null},
+    "yaxis6": {"scaleanchor": null},
+    "yaxis7": {"scaleanchor": null},
+    "yaxis8": {"scaleanchor": null},
```

Comment:
Can we use `false` instead of `null`?

Reply:
Or even better `true` instead of `null`?

Reply:
We should :hocho: all these lines instead.

Reply:
Done in d71c3b345d2eecb7b3087c383f00f5f102b9c2f6 for all the mocks